server/controllers/users.js

```diff
@@ -60,20 +65,26 @@ class UserController {
       const errorValue = error.details[0].message.replace(/\"/g, '');
       return res.status(400).json({ status: 400, error: errorValue });
     }
-    passport.authenticate('local', { session: false }, (
-      err,
-      user,
-      info
-    ) => {
-      if (err) {
-        return next(err);
-      }
-
-      if (user) {
-        return res.json({ user });
-      }
-      return res.status(400).json(info);
-    })(req, res, next);
+    try {
```

Comment:
A validator was created for input (see sign up) I expect that you make use of it.
You may need to modify the values sync with @despeauxz

Reply:
It has been implemented.